pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>org.disq-bio</groupId>
     <artifactId>disq</artifactId>
-    <version>0.3.7-SNAPSHOT</version>
+    <version>0.3.7-bugfix-SNAPSHOT</version>
```

Comment:
This change isn't necessary for this pull request -- versioning will be handled by the Maven release plugin on the next release cycle.

Reply:
Woops!  That's funny, I literally told @tedsharpe the same thing on his last PR and then I did it myself...